scripts/run_e2e_tests.sh

```diff
@@ -25,6 +25,25 @@ else
   CYPRESS_RECORD=""
 fi
 
+# Wait for a service to be up
+function wait_for_service() {
+  printf "> Waiting for %s to be ready... " "$1"
+  while true; do
+    nc -z "$2" "$3"
+    EXIT_CODE=$?
+    if [ $EXIT_CODE -eq 0 ]; then
+      printf "Application is up!\n"
```

Comment:
I would have added %s in the second message.

Reply:
I'm going to fix that, the way it logged on my local machine was different. This "Application is up!" on CircleCi is confusing.